plugins/Annotations/API.php

```diff
@@ -208,12 +246,12 @@ public function getAll(string $idSite, ?string $date = null, string $period = 'd
      * @param string $idSite The site ID(s) to get the annotation count for.
      * @param string $date The date of the period.
      * @param string $period The period type.
-     * @param int|bool $lastN Whether to get counts for the last N number of periods or not.
+     * @param int|null $lastN Whether to get counts for the last N number of periods or not.
      * @param bool $getAnnotationText
      * @return array An array mapping site IDs to arrays holding dates & the count of
      *               annotations made for those dates. eg,
      *               array(
-     *                 5 => array(
+     *                  => array(
```

Comment:
```suggestion
     *                 5 => array(
```

Looks like an accidental change.

Reply:
Reverted